e2e/cypress/support/ui_commands.js

```diff
@@ -421,3 +421,111 @@ Cypress.Commands.add('fileUpload', (targetInput, fileName = 'mattermost-icon.png
         );
     });
 });
+
+/**
+ * Upload a file on target input in binary format -
+ * @param {String} Subject - cypress/fixtures/matterMost.tar.gz
+ * @param {String} filePath - cypress/fixtures/matterMost.tar.gz
+ * @Param {String} fileType - application/gzip
+ */
+Cypress.Commands.add('uploadFile2', {prevSubject: true}, (subject, fileName, fileType) => {
+    cy.log('Upload process started .FileName:' + fileName);
+    cy.readFile(fileName, 'binary').then((content) => {
```

Comment:
You may use `cy.fixture` so that you may pass filename instead of relative path.

Reply:
Cy.fixture doesn't support tar.gzip files.initially tha plan was to use this but cypress has this restrictions had to use readfile option

Reply:
Not sure why but looking into it again, it probably didn't work on you since it's used incorrectly here - https://github.com/mattermost/mattermost-webapp/pull/4460/files#diff-ecaf15e463c339446fc523beb333c97aR512.
```diff
- .uploadFile2(fileName, fileName, fileType)
+ .uploadFile2(fileName, fileType) // correct usage
```

My suggestion would be:
```javascript
Cypress.Commands.add('uploadFile', {prevSubject: true}, (targetInput, fileName = 'mattermost-icon.png', mimeType = 'image/png', encoding) => {
    cy.fixture(fileName, encoding).then((content) => {...});
});
```
In your case, you may use it as 
```javascript
.uploadFile(fileName, mimeType, 'binary')
```
I think we'll be using this command to replace the existing `cy.fileUpload`, like
```diff
- cy.fileUpload('#fileUploadInput');
+ cy.get('#fileUploadInput').uploadFile();
```

Reply:
I tried using the same it didn't worked for Tar File .Will double check on this

Reply:
cy.fixture load is working fine . Overlooked at the parameters. Updated the testcase